<launch>    
	<!-- Arguments -->
	<arg name="filename" default="$(find occupancy_map_visualizer)/samples/octree.bt" />

    <!-- Parameters -->
    <param name="fixed_frame_id"	value="map" />
	<param name="octomap_filename"	value="$(arg filename)" />

    <!-- Run Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find occupancy_map_visualizer)/rviz/octomap.rviz" />

    <!-- Octree Visualization -->
    <node pkg="occupancy_map_visualizer"	type="octomap_visualizer_node"	name="octomap_visualizer_node"	output="screen"/>
</launch>
